src/rgw/rgw_admin.cc

```diff
@@ -2410,7 +2409,6 @@ int main(int argc, const char **argv)
       break;
     } else if (ceph_argparse_flag(args, i, "-h", "--help", (char*)NULL)) {
       usage();
-      return 0;
```

Comment:
Shouldn't these return statements be kept to actually stop further processing in this method?